Cargo.toml

```diff
@@ -8,6 +8,7 @@ keywords = ["COSE"]
 categories = ["cryptography"]
 repository = "https://github.com/awslabs/aws-nitro-enclaves-cose"
 description = "This library aims to provide a safe Rust implementation of COSE, with COSE Sign1 currently implemented."
+rust-version = "1.57"
```

Comment:
If there is a need to do a version bump, I'd recommend aligning with the MSRV of the [aws-sdk-rust](https://github.com/awslabs/aws-sdk-rust), which is **1.58.1.**